### PR TITLE
tap: support --quieter flag

### PR DESCRIPTION
**Pull request by xu-cheng**
cc @mikemcquaid

### CODE REVIEW

**Comment by mikemcquaid** [comment]
:+1:

File: Library/Homebrew/cmd/tap.rb [hunk 1]
```diff
@@ -16,8 +16,9 @@ def tap
     else
       tap = Tap.fetch(ARGV.named[0])
       begin
-        tap.install(:clone_target => ARGV.named[1],
-                    :full_clone   => ARGV.include?("--full"))
+        tap.install :clone_target => ARGV.named[1],
+                    :full_clone   => ARGV.include?("--full"),
+                    :quiet        => ARGV.quieter?
```
Comment (UniqMartin) [inline Library/Homebrew/cmd/tap.rb]: Should that have been documented in the `man` page?

Comment (xu-cheng) [inline Library/Homebrew/cmd/tap.rb]: Good idea.

**Comment by UniqMartin** [comment]
Not objecting to this change, but it would have been great to see some motivation for this addition in the commit message and/or the top comment in this PR. (The “what” is already evident from the code, but the “why” is totally unclear to me.)

**Comment by xu-cheng** [comment]
> but the “why” is totally unclear to me.

Sorry, the recent PRs I created are all to prepare the core/formula separation. You can get more detail from Slack chat history.

As for this PR, we may need to tap new core tap without creating any output noise for certain commands  e.g. `brew info --json=v1 `

**Comment by UniqMartin** [comment]
> Sorry, the recent PRs I created are all to prepare the core/formula separation. You can get more detail from Slack chat history.

Sorry if I'm being annoying here, but I really think an outsider should be able to understand this without having access to this chat history (which is volatile anyway and soon no longer accessible even to team members). Already the single short sentence you now wrote above makes it a lot easier to understand this change. Thanks for that!

**Comment by xu-cheng** [comment]
The fault is mine. I guess I don't write too many comment as a result that English is not my native language.

**Comment by UniqMartin** [comment]
No worries! It's not my native language either (though it's probably still easier for me as it is a more closely related language), thus I understand that it is a non-trivial effort to write some good comments and/or documentation in addition to the code.

**Pull request closed without merging**